keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -149,24 +149,19 @@ func GetOSInterface() types.OSTypeInstaller {
 
 // RunningModuleV2 identifies cloudcore/edgecore running or not.
 // only used for cloudcore container install and edgecore binary install
-func RunningModuleV2(opt *types.ResetOptions) (types.ModuleRunning, error) {
+func RunningModuleV2(opt *types.ResetOptions) types.ModuleRunning {
 	osType := GetOSInterface()
-	cloudCoreRunning, err := IsCloudcoreContainerRunning(constants.SystemNamespace, opt.Kubeconfig)
-	if err != nil {
-		return types.NoneRunning, err
-	}
+	cloudCoreRunning, _ := IsCloudcoreContainerRunning(constants.SystemNamespace, opt.Kubeconfig)
 	if cloudCoreRunning {
-		return types.KubeEdgeCloudRunning, nil
+		return types.KubeEdgeCloudRunning
 	}
 
-	edgeCoreRunning, err := osType.IsKubeEdgeProcessRunning(KubeEdgeBinaryName)
+	edgeCoreRunning, _ := osType.IsKubeEdgeProcessRunning(KubeEdgeBinaryName)
```

Comment:
print the err?

Reply:
if there is a real err when check cloudcore/edgecore  running,  ignore the err may have problem, we should recommend user to have retry